src/encoded/static/components/summary.js

```diff
@@ -297,6 +297,7 @@ class SummaryVerticalFacets extends React.Component {
 
         return (
             <div className="summary-content__facets-vertical">
+                <FilterList {...this.props} />
```

Comment:
This is purely a personal preference (and I might have mentioned it before anyway) but I prefer to use destructuring as properties to a component as rarely as possible, because I sometimes wonder where certain properties came from, and destructuring can obfuscate that. FilterList only takes one property anyway, so it would seem easy to just use that explicitly.

Reply:
OK, I think it is good now.